src/rsa.rs

```diff
@@ -504,6 +539,123 @@ mod tests {
     #[cfg(feature = "rsa_signing")]
     extern { static GFp_BN_BLINDING_COUNTER: u32; }
 
+    #[cfg(feature = "rsa_signing")]
+    #[test]
+    fn test_signature_rsa_pss_sign() {
+        struct DeterministicSalt<'a> {
```

Comment:
Potentially. I wasn't previously aware of `FixedSliceRandom`, which is nice. I wrote it in its current form so that it could be re-used in subsequent calls (I think it's used in RSA blinding). I'll see if I can just use `FixedSliceRandom` or `FixedSliceSequenceRandom`.